zizmor.yml

```diff
@@ -0,0 +1,7 @@
+# Configuration file for zizmor GitHub action checker
+rules:
+  template-injection:
+    ignore:
+      # Ignore the use of `runner.temp`, as we believe it to be safe
+      - hugo.yaml:34:9
+      - hugo.yaml:51:9
```

Comment:
Should this have a `\n` at the end of the file?

Reply:
zizmor does not require it, but I can add one for tidyness.